cloudant-client/build.gradle

```diff
@@ -0,0 +1,129 @@
+//note the gradle project name (cloudant-client) is the maven artifactId
+//the maven pom name entry and User-Agent name in client.properties are defined by clientName
+ext.clientName = 'java-cloudant'
+
+// Additional configurations for javadoc
+configurations {
+    linkableJavadoc {
+        transitive false
+    }
+}
+
+dependencies {
+    compile project(':cloudant-http')
+    linkableJavadoc project(':cloudant-http')
+    compile group: 'commons-codec', name: 'commons-codec', version: '1.6'
```

Comment:
commons-codec, junit, com.squareup.okhttp dependencies are all defined in top-level build.gradle, so do they need to be declared here?